php-tasks/requests.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+/**
```

Comment:
/**
 * Requests for Email data.
 *
 * @param string $url
 *   Url of the website the user will be providing.
 *
 * @return string
 *   Returns the user data.
 */